### PR TITLE
[DDW-302] Add acceptance tests for maximum wallets limit

### PR DESCRIPTION
**Pull request by daniloprates**
This PR adds acceptance tests for maximum wallets limit.

## Todo:

- [x] Create 20 wallets and checks if all of them are visible in the UI
- [x] Check if the buttons in the "Add wallet" screen are disabled
- [x] When deleting a wallet, the buttons should get enabled

---

### Basics

- [x] PR is updated to the most recent version of target branch (and there are no conflicts)
- [x] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [x] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`npm run test`)
- [x] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`npm run dev`)
- [x] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`npm run package` / CI builds)
- [x] There are no *flow* errors or warnings (`npm run flow:test`)
- [x] There are no *lint* errors or warnings (`npm run lint`)
- [x] Text changes are proofread and approved (Jane Wild)
- [x] There are no missing translations (running `npm run manage:translations` produces no changes)
- [x] UI changes look good in all themes (Alexander Rukin)
- [x] Storybook works and no stories are broken (`npm run storybook`)
- [x] In case of npm dependency changes both `package-lock.json` and `yarn.lock` files are updated

### Code Quality
- [x] Important parts of the code are properly documented and commented
- [x] Code is properly typed with flow
- [x] React components are split-up enough to avoid unnecessary re-rendering
- [x] Any code that only works in Electron is neatly separated from components

### Testing
- [x] New feature / change is covered by acceptance tests
- [x] All existing acceptance tests are still up-to-date
- [x] New feature / change is covered by Daedalus Testing scenario
- [x] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

File: features/wallets-limit.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  Scenario: User deletes one wallet and re-enable its Adding new wallets
+    Given I am on the "Wallet 20" wallet "settings" screen
+    When I click on delete wallet button
+    And I see delete wallet dialog
+    And I click on the "Make sure you have access to backup before continuing" checkbox
+    And I enter "Wallet 20" as name of the wallet to confirm
+    And I submit the delete wallet dialog
+    And I click on the add wallet button in the sidebar
+    Then The buttons in the Add Wallet screen should be enabled
```
Comment (DominikGuzei) [inline features/wallets-limit.feature]: Since this test is not really about "deleting a wallet" i would like the deletion to be shortened to a single step like this:

```gherkin
Scenario: User deletes one wallet and re-enable its Adding new wallets
  Given I create wallets until I reach the maximum number permitted
  When I delete the last wallet
  Then The buttons in the Add Wallet screen should be enabled
```

Comment (daniloprates) [reply features/wallets-limit.feature]: ✓
